crates/red_knot_python_semantic/resources/mdtest/conditional/match.md

```diff
@@ -3,40 +3,43 @@
 ## With wildcard
 
 ```py
-match 0:
-    case 1:
-        y = 2
-    case _:
-        y = 3
-
-reveal_type(y)  # revealed: Literal[2, 3]
```

Comment:
We are "too smart" now and infer `Literal[3]` here, as we can see that the first pattern can never match. To keep the original intent of the test, I changed the match target to an unknown `int`.